exemples/mem-place.json

```diff
@@ -0,0 +1,48 @@
+{
+	"kdbv": "~/Documents/repos/bsp0-qoriq-t1/build_core/kdbv",
+	"memreport": "~/Documents/repos/corunners-example/build/H/H/c0-on/gen/app/applink/memreport_out.ks",
+	"default_kmemory": "~/Documents/repos/corunners-example/build/H/H/c0-on/gen/app/applink/kmemory_out.json",
+	"out_kmemory": "~/Documents/repos/corunners-example/build/H/H/c0-on/kmemory_out.json",
```

Comment:
I'm not sure what purpose serves this file. Isn't this generated from a template?

Reply:
This is an exemple of what the generated file could be. It is not used in the scripts, only cited [here](https://github.com/charlie45000/corunners-example/blob/pgi/to_p2020/scripts/gen-kmem.py#L12)